tests/python/relay/test_pass_to_a_normal_form.py

```diff
@@ -94,6 +95,25 @@ def test_if():
     assert tvm.ir.structural_equal(anf, expected_output)
 
 
+def test_let_as_subexpr():
```

Comment:
Here's where you could throw an on_device annotation in.
  on_device(expr, virtual_device=..., constrain_result=True)